src/impact-dashboard/CurveChartContainer.tsx

```diff
@@ -48,7 +60,9 @@ const CurveChartContainer: React.FC<Props> = ({
   }, [groupStatus, curveData]);
 
   return !curveDataFiltered ? (
-    <Loading />
+    <LoadingContainerCustom>
```

Comment:
For consistency sake, I think it would be best to keep the `<Loading />` component here and just pass in some custom styles. For an example of how to do this you can look at `InputButton.tsx`. The `margin-top` is a conditional style based on the props. You can do something like that for the min-height and padding-bottom here. Does that make sense

Reply:
@jovergaag thanks for the example! Will give that a try and report back

Reply:
@jovergaag thanks that was super helpful, learned something! 🙏

I was trying to follow https://github.com/Recidiviz/covid19-dashboard/blob/master/src/design-system/Spacer.tsx or https://styled-components.com/docs/basics#extending-styles before 

let me know what you think